demos-europe/demosplan-core
# fix(refs DPLAN-12746): pass tooltipOptions via new prop

### Ticket

[DPLAN-12746](https://demoseurope.youtrack.cloud/issue/DPLAN-12746/Hinzufugen-von-Textbaustein-zu-Mitteilung-Invalid-Prop)



The `DpContextualHelp` `text` prop was used to pass an object with v-tooltip options, but it expected a string. In https://github.com/demos-europe/demosplan-ui/pull/1084, a new prop `tooltipOptions` was added that can be used to pass both text and other tooltip options. It is now used here to prevent the console error.

### How to review/test


Go to the 'Aktuelles' view and create a new message. Try to add a boilerplate text via the text editor. There should be no error in the console.

### Linked PRs

- [x] https://github.com/demos-europe/demosplan-ui/pull/1084

### Tasks (optional)

- [x] Merge https://github.com/demos-europe/demosplan-ui/pull/1084
- [ ] Release new demosplan-ui version

### PR Checklist
